pydoctor/templatewriter/pages/__init__.py

```diff
@@ -469,7 +465,7 @@ def objectExtras(self, ob: model.Documentable) -> List[Tag]:
         page_url = self.page_url
         name = ob.name
         r: List[Tag] = []
-        for b in self.ob.allbases(include_self=False):
+        for b in self.ob.mro():
```

Comment:
```suggestion
        for b in self.ob.mro()[1:]:
```
same comment as above.